lib/components/card_deck.dart

```diff
@@ -0,0 +1,378 @@
+import 'package:flutter/material.dart';
+import 'package:literature/models/playing_cards.dart';
+import 'package:enum_to_string/enum_to_string.dart';
+
+class CardDeck extends StatefulWidget {
+  CardDeck({
+    Key key,
+    this.cards,
+  }): super(key: key);
+
+  List<PlayingCard> cards;
+
+  _CardDeckState createState() => _CardDeckState();
+}
+
+class _CardDeckState extends State<CardDeck> {
+  static const font = "Courier New";
+
+  // Do we have to show
+  // the miniView?
+  bool _miniView = false;
+
+  // IMPORTANT: Container height
+  // See: card height.
+  double height = 200.0;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  void dispose() {
+    super.dispose();
+  }
+
+  Widget _largeCardList() {
+    return _miniCardsDeck();
+  }
+
+  Widget _rowCardsView() {
+    return _largeCardsDeck();
+  }
+  
+  @override
+  Widget build(BuildContext context) {
+    return new Container(
+      color: Colors.transparent,
+      height: height,
+      child: new Column(
+        children: <Widget>[
+          new Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              SizedBox(
+                height: 20,
+                width: 84,
+                child: new RaisedButton(
+                  onPressed: () {
+                    if (height == 320.0) {
```

Comment:
Giving hard values might be protruding in smaller devices, using media query instead will be great.
https://www.youtube.com/watch?v=-ZsWtafLLWg